modules/getting-started/pages/hargrave-writing-sample.adoc

```diff
@@ -0,0 +1,43 @@
+= Transportation to Salesforce Tower
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:noindex:
+
+In San Francisco, you have several transportation options for getting to Salesforce Tower:
+
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#rideshare-service[Rideshare service]
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#bus[Bus]  
+* https://github.com/mulesoft/docs-general/new/latest/modules/getting-started/pages#personal-vehicle[Personal vehicle] (car, motorcycle, bicycle)
+
+== Rideshare service
+
+Rideshare service is the most convenient transportation option if you have a smartphone and cost is not an issue.
+
+To get to Salesforce Tower using a ride share service:
```

Comment:
rideshare or ride share or ride-share? Should be consistent?

Reply:
Good catch - thank you! I will update this to be 'rideshare'.

Reply:
Was going to fix this to be consistent ('rideshare'), but it looks like editing is disabled.